lib/elixir/test/elixir/test_helper.exs

```diff
@@ -1,4 +1,10 @@
-ExUnit.start [trace: "--trace" in System.argv]
+exclude =
+  case :erlang.system_info(:otp_release) do
```

Comment:
@fishcakez when the fix is released we would remove the exclusion? Or again, we could use system_info(:version), but I'm not sure that's guaranteed to increment with new minor releases.